index.js

```diff
@@ -0,0 +1,51 @@
+const express = require('express');
+const cors = require('cors');
+const bcrypt = require('bcryptjs');
+
+const db = require('./data/dbHelpers.js');
+
+const server = express();
+
+server.use(express.json())
+server.use(cors())
+
+const PORT = 4400;
+
+server.get('/api/users', (req, res) => {
+    db.getUsers()
+        .then(users => {
+            res.json(users)
+        })
+        .catch(err => {
+            res.send(err)
+        });
+});
+
+server.post('/api/register', (req, res) => {
+    const user = req.body;
+    user.password = bcrypt.hashSync(user.password, 14);
+    db.insertUser(user)
+        .then(ids => {
+            res.status(201).json({ id: ids[0] });
```

Comment:
A couple of other options for getting this first id (`ids[0]`):

- `first` method to grab the first element of an array
```javascript
db.insertUser(user)
  .first()
  .then(id => {
    res.status(201).json({ id });
    // ...
```
- array destructuring in the parameter list to get the first element
```javascript
db.insertUser(user)
  .then(([ id ]) => {
    res.status(201).json({ id });
    // ...
```